### DISCUSSION TITLE
G Helper update procedure

### DISCUSSION THREAD

**Discussion by Chrystef** [seerge/g-helper#1377] [Q&A] [answered]
Greetings Dear all,

I search a bit but did not found a clear answer..in order to update g helper **without losing my settings** do i just replace the .exe file? or all the folder with the .config .dll .sys ?

Thank you in advance

**Comment by seerge** [comment, marked as answer]
@Chrystef hello, you don't lose settings when you update the app as [they are stored separately ](https://github.com/seerge/g-helper#where-can-i-find-app-settings-or-logs-)
